content/en/docs/tasks/observability/logs/access-log/index.md

```diff
@@ -97,6 +97,6 @@ $ kubectl delete -f @samples/httpbin/httpbin.yaml@
 Edit the `istio` configuration map and set `accessLogFile` to `""`.
 
 {{< text bash >}}
-$ helm template install/kubernetes/helm/istio --namespace=istio-system -x templates/configmap.yaml | kubectl replace -f -
+$ istioctl manifest apply
```

Comment:
TODO sort out this operation - this isn't correct as is.

Reply:
Are you sure this isn't correct. With the explicit --set accessLogFile arg, won't it switch back to the default ""?